### PR TITLE
fixing incorrect use of limit

### PR DESCRIPTION
**Pull request by iamkhush**
Here were are not using the `Limit` parameter to limit the results loaded.
I think this is an appropriate fix

### CODE REVIEW

**Comment by iamkhush** [comment]
Is the pull request clear about its purpose?

File: src/autocomplete.ts [hunk 1]
```diff
@@ -348,20 +348,22 @@ class AutoComplete {
         _RenderResponseItems: function(response: ResponseItem[]): HTMLElement {
             var ul: HTMLElement = document.createElement("ul"),
                 li: HTMLElement = document.createElement("li");
-            
+
+            var limit = this._Limit();
```
Comment (baptistedonaux) [inline src/autocomplete.ts]: Could you typed your parameter (```: number```) and merge this variable with previous.

**Comment by baptistedonaux** [comment]
Hi and sorry my "brain-latency" :-p 

Thank you for your fix ! I have just suggest you a little modification.

**Comment by iamkhush** [comment]
Hey, looks like its alright now.

**Review by baptistedonaux — CHANGES_REQUESTED** [review]

File: src/autocomplete.ts (hunk 1 above)
Comment (baptistedonaux) [reply src/autocomplete.ts]: Sorry, I had missing to send my request changes.

Comment (iamkhush) [reply src/autocomplete.ts]: Do you mean `var limit = Number(this._Limit())` ?

Comment (baptistedonaux) [reply src/autocomplete.ts]: Fixed into https://github.com/autocompletejs/autocomplete.js/pull/32/commits/3cf6bcfc90600112298044400177ec19015d89c3

**Review by baptistedonaux — APPROVED** [review]

**Comment by baptistedonaux** [comment]
```v2.1.4``` just released ! Thanks for your fixes !

**Pull request merged**